packages/ui-billboard/src/Billboard/__tests__/Billboard.test.tsx

```diff
@@ -42,7 +43,10 @@ describe('<Billboard />', async () => {
   })
 
   describe('with generated examples', async () => {
-    generateA11yTests(Billboard, BillboardExamples)
+    generateA11yTests(
+      Billboard,
+      BillboardExamples as StoryConfig<BillboardProps>
```

Comment:
You can put the `as StoryConfig<BillboardProps>` cast at the end of the export at `Billboard.examples.tsx` too. That way it is already exported as a StoryConfig and you don't have to use the cast here :)

Reply:
Great idea! I will change it.